### DISCUSSION TITLE
How to delete publishing page?

### DISCUSSION THREAD

**Discussion by dag7dev** [alshedivat/al-folio#253] [Q&A] [answered]
I currently don't have any publication, how can I hide that link in the homepage?

**Comment by alshedivat** [comment, marked as answer]
Just remove `publications.md` from `_pages/`.

**Comment by dag7dev** [comment]
Oh it was so easy!
Thank you!

**Comment by fl-yu** [comment]
How to hide/remove blog section? I did not find .md in _pages/. Thank you!!!

**Comment by loodvn** [comment]
I also just moved stuff like cv.md, projects.md, dropdown.md out to a folder _pages_unused/ as I'd like to possibly add them later and don't want to remember to cross-reference the original template the whole time :)
